unlock-protocol.com/blog/protocol-for-membership.md

```diff
@@ -0,0 +1,34 @@
+---
+title: A Protocol for Membership
+subTitle: Monetizing creation by leveraging membership to communities
+authorName: Julien Genestoux
+publishDate: June 19, 2019
+description: The unlock protocol provides a way for people to show their support and access exclusive content, features, or even status. It is a web-native way for creators to monetize their work.
+image: /static/images/blog/protocol-for-memberships.png
+---
+
+The difference between the web and native applications is the presence of **links**. These links are the reason why the web is a "network" as they connect pages together, making a graph. Google and its pagerank algorithm have leveraged that graph to rank wesbites, by assuming that the sites which have the most links are the most relevant.
+
+![Use one Unlock lock for a variety of services and products.](/static/images/blog/hero.jpg)
+
+## Memberships > Attention
+
+However, since the begining, these links have also formed **communities**, as groups of sites linking to one another. Websites which discuss the same topics tend to link to each other, creating de-facto communities. The social web re-inforced that trend: people following each other formed more communities. Creators gathered groups of users around their work. Whether they are blogger or journalists, musician or even developers, they formed groups which attracted users because of their values, the content they produce, or even the tools that were offered to the members of the community.
```

Comment:
```suggestion
However, since the beginning, these links have also formed **communities**. Websites which discuss the same topics tend to link to each other, creating de-facto communities of interest. The social web reinforced that trend: people following each other formed more communities. Creators gathered groups of users around their work. Whether they are blogger or journalists, musician or even developers, they formed groups which attracted users because of their values, the content they produce, or even the tools that were offered to the members of the community.
```